packages/apps/kadena-docs/src/components/Markdown/Figure/Figure.tsx

```diff
@@ -0,0 +1,33 @@
+import { Text } from '@kadena/react-components';
+
+import { StyledFigure } from './styles';
+
+import Image from 'next/image';
+import React, { FC, ReactNode } from 'react';
+
+interface IProps {
+  children?: ReactNode;
+  alt: string;
+  src: string;
+}
+
+export const Figure: FC<IProps> = ({ children, alt, src }) => {
+  return (
+    <StyledFigure>
+      {children}
+      <Image
+        src={src}
+        alt={alt}
+        width="0"
+        height="0"
+        sizes="100vw"
```

Comment:
do we need to add media queries here for different breakpoints?

Reply:
nope..this way it is just responsive.
or when it is smaller, it will show the actual size